src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -1,19 +1,17 @@
 import { Text } from '@konturio/ui-kit';
 import { i18n } from '~core/localization';
-import errorImage from './error-state-img.png';
 import s from './ErrorMessage.module.css';
 
 export function ErrorMessage({
   message,
-  marginTop = '30%',
+  margin = '32px 16px',
 }: {
   message?: string;
-  marginTop?: string;
+  margin?: string;
 }) {
   return (
-    <div className={s.spinner} style={{ marginTop }}>
+    <div className={s.spinner} style={{ margin }}>
```

Comment:
Could you please rename this style to something more appropriate? This is definitely not a spinner (probably an old copypaste which nobody fixed)